src/palace/manager/scripts/lexile_db.py

```
@@ -0,0 +1,46 @@
+from palace.manager.celery.tasks.lexile import lexile_db_update_task
+from palace.manager.scripts.base import Script
+
+
+class LexileDBUpdateScript(Script):
+    """Kick off the Lexile DB update task."""
+
+    name = "Lexile DB Update"
+
+    def __init__(
+        self,
+        force: bool = False,
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)